src/main/java/frc/robot/constants/ShooterConstants.java

```diff
@@ -6,21 +6,4 @@ public class ShooterConstants {
     public static final int BOTTOM_MOTOR_ID = 99;
     public static final int TOP_MOTOR_ID = 99;
 
-    // constants for Pids
-    public static final double TOP_P = 0.00005;
-    public static final double TOP_I = 0;
-    public static final double TOP_D = 0;
-
-    public static final double S = 0;
-    public static final double V = 1.0/6000;
-
-    public static final double BOTTOM_P = 0.00005;
-    public static final double BOTTOM_I = 0;
-    public static final double BOTTOM_D = 0;
-
-    /**
-     * In RPM
-     */
-    public static final double TOLERANCE = 10;
-
```

Comment:
This should probably still be here. It makes tuning easier if constants are easier to find and in 1 spot. Also, do there need to be separate PID constants for the top and bottom? They have the same type of motor, gear ratio, and wheels. 2 PIDControllers make sense, but they could use the same constants.

Reply:
tolerance is there, i dont know why it appeared as removed. not too sure about the PID constants thing, you should ask someone more experienced in tuning PID and PID overall